src/main/java/frc/robot/subsystems/gpm/Intake.java

```diff
@@ -142,8 +139,8 @@ public void simulationPeriodic() {
         flywheelSim.update(0.020);
         centeringFlywheelSim.update(0.020);
 
-        motorRPMSim = flywheelSim.getAngularVelocityRPM();
-        centeringMotorRPMSim = centeringFlywheelSim.getAngularVelocityRPM();
+        flywheelSim.getAngularVelocityRPM();
+        centeringFlywheelSim.getAngularVelocityRPM();
 
```

Comment:
New lines 142 and 143 have no effect. They were values that would be stored in motorRPMSim and centeringMotorRPM Sim, but those variables have been removed.